### ISSUE TITLE
Process finished with exit code 139 (interrupted by signal 11: SIGSEGV)

### ISSUE THREAD

**Issue by YingjieYin** [NVIDIA/flownet2-pytorch#78]
when i run the code , it will give the following error:
Parsing Arguments
  [0.021s] batch_size: 8
  [0.021s] crop_size: [256, 256]
  [0.021s] fp16: False
  [0.021s] fp16_scale: 1024.0
  [0.021s] gradient_clip: None
  [0.021s] inference: False
  [0.021s] inference_batch_size: 1
  [0.021s] inference_dataset: MpiSintelClean
  [0.021s] inference_dataset_replicates: 1
  [0.021s] inference_dataset_root: ./MPI-Sintel/flow/training
  [0.021s] inference_n_batches: -1
  [0.021s] inference_size: [-1, -1]
  [0.021s] log_frequency: 1
  [0.021s] loss: L1Loss
  [0.021s] model: FlowNet2
  [0.021s] model_batchNorm: False
  [0.021s] model_div_flow: 20.0
  [0.021s] name: run
  [0.021s] no_cuda: False
  [0.021s] number_gpus: 1
  [0.021s] number_workers: 8
  [0.021s] optimizer: Adam
  [0.021s] optimizer_amsgrad: False
  [0.021s] optimizer_betas: (0.9, 0.999)
  [0.021s] optimizer_eps: 1e-08
  [0.021s] optimizer_lr: 0.0001
  [0.021s] optimizer_weight_decay: 0
  [0.021s] render_validation: False
  [0.021s] resume: /path/to/checkpoints
  [0.021s] rgb_max: 255.0
  [0.021s] save: ./work
  [0.021s] save_flow: False
  [0.021s] schedule_lr_fraction: 10
  [0.021s] schedule_lr_frequency: 0
  [0.021s] seed: 1
  [0.021s] skip_training: False
  [0.021s] skip_validation: False
  [0.021s] start_epoch: 1
  [0.021s] total_epochs: 10000
  [0.021s] train_n_batches: -1
  [0.021s] training_dataset: MpiSintelFinal
  [0.021s] training_dataset_replicates: 1
  [0.021s] training_dataset_root: /media/yyj/SuperDisk/DataSets/MPI-Sintel-complete/training
  [0.021s] validation_dataset: MpiSintelClean
  [0.021s] validation_dataset_replicates: 1
  [0.021s] validation_dataset_root: /media/yyj/SuperDisk/DataSets/MPI-Sintel-complete/training
  [0.021s] validation_frequency: 5
  [0.021s] validation_n_batches: -1
  [0.021s] Operation finished

Source Code
Initializing Datasets
  [0.035s] Training Dataset: MpiSintelFinal
  [0.084s] Training Input: [3, 2, 256, 256]
  [0.395s] Training Targets: [2, 256, 256]
  [0.626s] Validation Dataset: MpiSintelClean
  [0.850s] Validation Input: [3, 2, 256, 256]
  [1.077s] Validation Targets: [2, 256, 256]
  [1.077s] Operation finished

Building FlowNet2 model
  [4.326s] Effective Batch Size: 8
  [4.327s] Number of parameters: 162518834
  [4.327s] Initializing CUDA
  [6.202s] Parallelizing
  [6.203s] Random initialization
  [6.203s] Initializing save directory: ./work
  [6.205s] Operation finished

Initializing Adam Optimizer
  [0.001s] lr = 0.0001 (<type 'float'>)
  [0.001s] amsgrad = False (<type 'bool'>)
  [0.001s] weight_decay = 0 (<type 'int'>)
  [0.001s] betas = (0.9, 0.999) (<type 'tuple'>)
  [0.001s] eps = 1e-08 (<type 'float'>)
  [0.001s] Operation finished

Overall Progress:   0%|                                                   | 0/10000 [00:00<?, ?it/s]
Validating Epoch 0:   0%|                                                 | 0/130.0 [00:00<?, ?it/s]
Process finished with exit code 139 (interrupted by signal 11: SIGSEGV)

**Comment by TomHeaven** [comment]
I met the same problem using pytorch 1.0 and CUDA 10.

**Comment by Dceaser** [comment]
@TomHeaven @YingjieYin 
Hello, have you solved this difficulty? Could you please help me, I have also met this problem when I using torch.
Thanks!!!
